mllib/src/main/scala/org/apache/spark/ml/pmml/PMMLExportable.scala

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
Seems this is a copy-paste of org.apache.spark.mllib.pmml, should we deprecate the mllib one, and use the new one in ml package?